requirements-dev.txt

```diff
@@ -1,7 +1,9 @@
 behave==1.2.6
 behaving==1.5.6
-flake8==3.7.9
+flake8==3.8.3
 nose==1.3.7
 splinter>=0.13.0
+-e git+https://github.com/qld-gov-au/ckanext-data-qld@develop#egg=ckanext-data_qld
```

Comment:
is data used in publications?

Reply:
Parts of it are necessary for the theme, yes. I've broken up the extension into pieces, so that Publications can reuse the useful parts without incorporating the parts that depend on the existence of other extensions like Scheming.